configuration.k

```diff
@@ -0,0 +1,220 @@
+                    <Constructor> false </Constructor>
+                    // ... more to be added later ...
+                
+                </Contract>
+            </contracts>
+
+        </solidity>
+
+        // Ethereum Network
+        // ==========================
+
+        <network>
+
+            // Accounts Record
+            // ---------------
+
+            <activeAccounts> .Set </activeAccounts>
+	        <cntAccount> 0:Int </cntAccount>
+            <accounts>
+                <account multiplicity="*">
+                    
+                    // Same as in KEVM / IELE
+                    
+                    <acctID>  .K                      </acctID>
+                    <balance> 0                      </balance>
+		            <context> .Map		     </context>
+                    
+                    // When a contract is deployed, its <contract> data structure will be copied here.
+                    // Because contracts can create other contracts (via "new") we need to also keep the 
+                    // set of contract that this contract can create (i.e. the set of contracts that was
+                    // available at creation time, like in IELE)
+
+                    <code> 
+                        <associatedContract> .K </associatedContract>    // contract deployed on this account
+                        <availableContracts> .K </availableContracts>    // contracts that can be created by this account's contract
+                    </code>
+
+                    // [Jerry's notes about storage]:
+                    // this is exactly same as evm
+                    // Solidity has rules to allocate indices/offset(packing) to each storage
+                    // variable and I think these rules should be the part of Solidity semantics,
+                    // so that the storage usage of solidity program and evm bytecode have
+                    // the same effect on the storage.
+                    
+                    <Storage> .Map  </Storage>
+
+                    // Q. Why include allocation rules in solidity semantics?
+                    // A. 
+                    // * These rules are explained in the offical solidity docs
+                    // https://solidity.readthedocs.io/en/v0.4.24/miscellaneous.html?highlight=pack#layout-of-state-variables-in-storage
+                    // * AFAIK, allocation is not affected by optimization level.
+                    // * some contracts make use of this allocation rule to implement some nice stuff e.g.
+                    // https://github.com/gnosis/safe-contracts/blob/d279480b0be2562aab7dd0a751113ba8681e816e/contracts/common/MasterCopy.sol
+                    // https://github.com/gnosis/safe-contracts/blob/d279480b0be2562aab7dd0a751113ba8681e816e/contracts/proxies/Proxy.sol
+                    //
+
+                    // not needed, maybe?
+                    <nonce>   .K             </nonce>
```

Comment:
this is definitely necessary.

Reply:
Why is the nonce sort `.K`? Should it just be `0` so that it's sort `Int`?

Reply:
We are not sure if the `Int` sort in K framework is sufficient for `nonce` because the length of a nonce is 64-bit. Temporarily, we make it of the sort `.k`, just like the `acctID` cell for account ID.

Reply:
K's Int is unbounded so it should be ok to use them if we add some side conditions for the range of the value in the rules using nonce.